pytheas-core/src/main/java/com/netflix/explorers/providers/FreemarkerTemplateProvider.java

```diff
@@ -68,13 +75,13 @@ public static void addUrlHandler(String protocol, URLStreamHandler handler) {
     
     private Configuration fmConfig = new Configuration();
     private boolean servletMode = false;
-    
-    @Inject(optional=true) ExplorerManager manager;
+    private ExplorerManager manager;
 
     @Context 
     private ThreadLocal<HttpServletRequest> requestInvoker;
-
-    public void setExplorerManager(ExplorerManager manager) {
+    
+    @Inject
+    public FreemarkerTemplateProvider(ExplorerManager manager) {
```

Comment:
@Inject(optional=true) is not the right thing to do here and is what allowed the bug to go uncaught in the first place. If the field never gets set the error gets deferred until the first request is made instead of failing with a meaningful error at startup or compile time.